assets/tenancy_test_case.php.stub

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Tests;
+
+use Stancl\Tenancy\Tenant;
+use Stancl\Tenancy\TenantManager;
+use Illuminate\Foundation\Testing\TestCase;
+
+abstract class TenancyTestCase extends TestCase
+{
+    use CreatesApplication;
+
+    public $createTenant = true;
+
+    /**
+     * Setup the test environment
+     * before each individual test
+     *
+     * @return void
+     */
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        config(['database.default' => 'sqlite']);
+        config(['tenancy.database.suffix' => '.sqlite']);
+
+        $this->artisan('migrate');
```

Comment:
migrate:fresh as the system-connection is not recreated with each test.

Could also unlink and recreate it and then run migrate